### DISCUSSION TITLE
训练表格识别模型，有没有基于SLANet_lcnetv2.yml的预训练模型参数

### DISCUSSION THREAD

**Discussion by liuzhipengchd** [PaddlePaddle/PaddleOCR#13320] [Q&A]
我只找到了基于[SLANet_ch.yml]9.3Mb的预训练模型：[ch_ppstructure_mobile_v2.0_SLANet_train.tar]，
paddle 有没有提供SLANet_lcnetv2 版本的模型，可以让试用的

**Comment by GreatV** [comment]
参考 https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/algorithm_table_slanet.md#1-%E7%AE%97%E6%B3%95%E7%AE%80%E4%BB%8B
